tests/test_models.py

```diff
@@ -138,6 +143,41 @@ def test_delete_an_order(self):
         order.delete()
         orders = Order.all()
         self.assertEqual(len(orders), 0)
+    
+    def test_cancel_order(self):
+        """It should change status of order to Cancelled"""
+        orders = Order.all()
+        self.assertEqual(orders, [])
+        order = OrderFactory()
+        order.create()
+        # Assert that it was assigned an id and shows up in the database with status Cancelled
+        self.assertIsNotNone(order.id)
+        orders = Order.all()
+        self.assertEqual(len(orders), 1)
+        order = orders[0]
+        order.status = "Cancelled"
+        order.update()
+        orders = Order.all()
+        self.assertEqual(order.status, "Cancelled")
+    
+    def test_incomplete_order(self):
```

Comment:
@bartconsedine This is the one I can't get to work. Errors because ENUM from the fuzzychoice line in factories doesnt contain incomplete. Do I need an if statement or for loop to throw the error?